workflows/repo-specific/helm-charts/bump-tf-chart-version.yml

```
@@ -0,0 +1,92 @@
+name: Bump Terraform Chart Version
+on:
+  release:
+    types: [published]
+jobs:
+  bump-terraform-chart-version:
+    name: Bump Terraform Chart Version
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout helm-charts source code'
+        uses: 'actions/checkout@v3'
+        with:
+          path: helm-charts
+      - name: 'Checkout terraform-aws-cloudquery source code'
+        uses: 'actions/checkout@v3'
+        with:
+          path: terraform-aws-cloudquery
+          repository: cloudquery/terraform-aws-cloudquery
+          token: ${{ secrets.GH_PAT }}
```

```suggestion
          token: ${{ secrets.GH_CQ_BOT }}
```

Can we use the bot token? I think `GH_PAT` is @yevgenypats's token

---

fixed